pkgs/by-name/fl/flaresolverr/package.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  makeWrapper,
+
+  chromium,
+  python3,
+  undetected-chromedriver,
+  xorg,
+
+  nixosTests,
+}:
+
+let
+  python = python3.withPackages (
+    ps: with ps; [
+      bottle
+      func-timeout
+      prometheus-client
+      selenium
+      waitress
+      xvfbwrapper
+
+      # For `undetected_chromedriver`
+      requests
+      websockets
+    ]
+  );
+in
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
Use `rec` instead of `(finalAttrs: `?

Reply:
`finalAttrs` are preferred over `rec`, see more about that [here](https://github.com/NixOS/nixpkgs/blob/d7705cea769f962a0c56a7f7fe1448cfbb2fda45/doc/stdenv/stdenv.chapter.md#fixed-point-arguments-of-mkderivation-mkderivation-recursive-attributes).

Reply:
Didn’t know that, then lgtm!